### PR TITLE
fix: more windows interop

### PR DESCRIPTION
**Pull request by richardschneider**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,11 +57,11 @@
     "async": "^2.5.0",
     "base32.js": "~0.1.0",
     "cids": "~0.5.2",
-    "interface-datastore": "~0.4.0",
     "datastore-core": "~0.4.0",
     "datastore-fs": "~0.4.0",
-    "datastore-level": "~0.6.0",
+    "datastore-level": "github:ipfs/js-datastore-level#windows-interop",
```
Comment (diasdavid) [inline package.json]: Change to use the actual version

File: package.json [hunk 2]
```diff
@@ -48,6 +48,7 @@
     "lodash": "^4.17.4",
     "memdown": "^1.4.1",
     "multihashes": "~0.4.12",
+    "multihashing-async": "^0.4.7",
```
Comment (diasdavid) [inline package.json]: Was this dep necessary?

File: package.json [hunk 3]
```diff
@@ -11,9 +11,9 @@
     "./src/default-options.js": "./src/default-options-browser.js"
   },
   "scripts": {
-    "test": "aegir test",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser",
+    "test": "aegir test --timeout 10000",
+    "test:node": "aegir test --target node  --timeout 10000",
+    "test:browser": "aegir test --target browser --timeout 10000",
```
Comment (diasdavid) [inline package.json]: Avoid global timeouts

File: package.json (hunk 1 above)
Comment (richardschneider) [reply package.json]: @diasdavid was not ready for a review yet.  One the travis CI builds keeps on timing out.

Comment (diasdavid) [reply package.json]: Roger!

**Comment by diasdavid** [comment]
Resolved the conflict with Master (package.json)

File: package.json (hunk 2 above)
Comment (richardschneider) [reply package.json]: yes, `test/blockstore-test` requires it.  Its a devDependency

File: package.json (hunk 3 above)
Comment (richardschneider) [reply package.json]: ok, will use `.timeout` on the bad test

**Comment by richardschneider** [comment]
@diasdavid seems that appveyor has placed us in queue; I've been waiting 30 mins.  If it ever completes, could you review and release.  I going off-line soon.

**Pull request merged**